lightning/src/ln/channelmanager.rs

```diff
@@ -9059,7 +9151,7 @@ where
 			.add_new_awaiting_invoice(
 				payment_id, expiration, retry_strategy, max_total_routing_fee_msat
 			)
-			.map_err(|_| Bolt12SemanticError::DuplicatePaymentId)?;
+			.map_err(|_| Bolt12RequestError::DuplicatePaymentId)?;
```

Comment:
Let's use `()` instead of `_` here and in other places where `.map_err` is used. That way, if the error type ever is changed we are forced to reconsider what the proper mapping should be.

Reply:
Okay, good to know!